pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, zig, mktemp }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+
+, zigDefaultBuildFlags ? "-Drelease-safe -Dcpu=baseline"
+, zigBuildFlags ? ""
```

Comment:
we can just remove this and use `$zigBuildFlags`

```suggestion
```

Reply:
Should we, because i assume some extra build flags may be required in some instances.

Reply:
I'm not sure what you mean by that

Reply:
Should we really remove `zigBuildFlags` because some zig packages may require additional flags for compilation.

Reply:
No, what I meant is we should remove the parameter `zigBuidlFlags` and just use `$zigBuildFlags` directly, since the default is an empty string anyway

Reply:
[this](https://github.com/NixOS/nixpkgs/pull/241741#discussion_r1254695008) is what I mean